independent-projects/bootstrap/app-model/pom.xml

```diff
@@ -14,6 +14,22 @@
     <name>Quarkus - Bootstrap - App Model</name>
 
     <dependencies>
+        <dependency>
+            <groupId>org.gradle</groupId>
+            <artifactId>gradle-core-api</artifactId>
+            <version>6.1.1</version>
```

Comment:
I didn't realize we'd introduce Gradle deps in the app-model. This is not a good thing. For Maven we have `quarkus-bootstrap-maven-resolver` that depends on app-model and adds the Maven deps. I think we should do the same for Gradle.

Reply:
This should be getting now into the launcher JAR. Have you noticed how much bigger it becomes with the Gradle part?

Reply:
Actually, we don't need this dependency anymore, we only need the `gradle-tooling-api`. But, yes I think externalizing it to a module could be a good idea.